src/aliases/manager/manager.go

```diff
@@ -0,0 +1,52 @@
+package aliasmanager
+
+import (
+	"context"
+
+	aliasent "github.com/consensys/quorum-key-manager/src/aliases/entities"
+	aliasstore "github.com/consensys/quorum-key-manager/src/aliases/store"
+)
+
+var _ aliasent.AliasBackend = &BaseManager{}
+
+type BaseManager struct {
+	db aliasstore.Database
+}
+
+func New(db aliasstore.Database) *BaseManager {
+	return &BaseManager{
+		db: db,
+	}
+}
+
+// Start does nothing as the DB client is already connected.
+func (m *BaseManager) Start(_ context.Context) error { return nil }
+
+// Stop does nothing as the DB client should be stopped outside of it.
+func (m *BaseManager) Stop(_ context.Context) error { return nil }
+func (m *BaseManager) Error() error                 { return nil }
+func (m *BaseManager) Close() error                 { return nil }
+
+func (m *BaseManager) CreateAlias(ctx context.Context, registry aliasent.RegistryName, alias aliasent.Alias) (*aliasent.Alias, error) {
+	return m.db.Alias().CreateAlias(ctx, registry, alias)
```

Comment:
Any reason why your use cases are not doing anything? You probably need permission checks and logging here. (same for all use cases)

Reply:
There were no permissions that I knew of when I started. I will add this as an issue.
What's the verbosity of the logging we want? We enter the func, we log it? We log only on error?

I just created a boilerplate that is accessible from the stores service.

Reply:
We need to create an issue about the authorizations for alias creation/access. The implications are big (rewriting the destination of an alias for another address is really dangerous)
#345